src/contexts/rootContext.js

```diff
@@ -0,0 +1,4 @@
+import { createContext } from 'react';
+
+const RootContext = createContext();
```

Comment:
we could have a hook for using the context value exported from this file and then use it everywhere instead of importing `useContext` and the context itself, something like: 
```jsx
export const RootContext = createContext()
export const useRootContext = () => useContext(RootContext)
```